src/olympia/tags/cron.py

```diff
@@ -1,31 +0,0 @@
-import cronjobs
-
-from olympia.addons.models import Addon
-from olympia.tags.models import AddonTag, Tag
-
-
-@cronjobs.register
-def tag_jetpacks():
-    # A temporary solution for singling out jetpacks on AMO.  See bug 580827
-    tags = (
-        ('jetpack', {
-            '_current_version__files__jetpack_version__isnull': False
-        }),
```

Comment:
cc'ing @wagnerand regarding the tagging during upload time just to be extra sure. I remember hearing something that some tags are relevant for review purposes but maybe that's just relevant in validation messages but I'm not sure.